components/atoms/form/search/components/search.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Search
+ *
+ */
+
+// React
+import { func, string, oneOf } from 'prop-types'
+
+// UI
+import { Button, useChange, Form, Input } from 'components'
+
+// Style
+import styled from 'styled-components'
+
+export const Search = ({ label, onSearch, placeholder, value, type }) => {
+  const INITIAL_STATE = {
+    query: value || ''
+  }
+
+  const [change, form] = useChange(INITIAL_STATE)
+  const { query } = form
+
+  return (
+    <Form submit={() => onSearch(query)}>
+      <StyledSearch>
+        <Input change={change} id='query' placeholder={placeholder} required={false} type={type} value={query} />
```

Comment:
The 'Format so lines are 80 chars' task means that lines of code that are longer than 80 characters should be split into multiple lines. 
In this case, the <Input .../> line is too long.

Reply:
@majid-amiri @kheruc In VSCode you can set 
  "editor.rulers": [
    80
  ],

I notice I do not have an .editorconfig in industtryui either I will add one - you will need the plugin for VSCode: https://github.com/editorconfig/editorconfig-vscode

Reply:
Thanks guys. They're fixed now.